frontend-react/src/__mocks__/SettingsMockServer.ts

```diff
@@ -1,16 +1,43 @@
 import { rest } from "msw";
 import { setupServer } from "msw/node";
 
+import { RSService } from "../config/endpoints/settings";
+
 export const senderSettingsUrl = (orgName: string, senderName: string) =>
     `/api/settings/organizations/${orgName}/senders/${senderName}`;
 
+export const testService: RSService = {
+    customerStatus: "testing",
+    name: "test-service",
+    organizationName: "test-org",
+    topic: "test-topic",
+};
+
 const handlers = [
     rest.put(
         `http://localhost:3000${senderSettingsUrl("abbott", "user1234")}`,
         (req, res, ctx) => {
             return res(ctx.status(200));
         }
     ),
+    rest.get(
```

Comment:
Anyone got a good way to consolidate these?? I _think_ I can just replace `senders` and `receivers` with `:serviceType` and never actually measure it in the response. Senders and Receivers follow the same data shape, but my concern is that the variable path would open the door to some hard-to-track testing issues down the line. So I left it as WET as I could here haha